front/public/locales/de/operationalStudies/manageTrainSchedule.json

```
@@ -0,0 +1,170 @@
+{
+  "addPacedTrain": "Add a service",
+  "addVia": "Wegepunkt hinzufügen",
+  "addVias": "Wegepunkte hinzufügen",
+  "addTrainSchedule": "Zug hinzufügen",
+  "addTrainSchedules": "Einen oder mehrere Züge hinzufügen",
+  "anyTrack": "Beliebiges Gleis",
+  "blocktype": "Streckenblockart",
+  "BoundsAreLinked": "Begrenzungen sind miteinander verknüpft",
+  "BoundsAreNotLinked": "Begrenzungen sind nicht miteinander verknüpft",
+  "returnToSimulationResults": "Zurück zu Simulationsergebnissen",
+  "cancelPathfinding": "Abbrechen",
+  "chooseOnMap": "Auf Karte auswählen",
+  "composition": "Zusammensetzung",
+  "coreErrors": {
+    "path_with_repeated_tracks": "Der Pfad verläuft mehrmals über dasselbe Gleis."
```

Same as above, if station-specific:

```suggestion
    "path_with_repeated_tracks": "Der Pfad verläuft mehrmals über dieselbe Strecke."
```